src/components/bluebutton.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './bluebutton.scss';
+
+const Button = ({ targetTo, children, label, ...props }) =>
```

Comment:
Exactly, why would you call this fuction `targetTo`?  I know it's copied from `Link`, and for link it sort-of made sense to call it `targetTo`. Here it doesn't make much sense.

This is a name you can choose, and you can choose it to be anything you want. You can keep the name `targetTo` but you can also change it into something else.